tap.go

```diff
@@ -68,4 +69,4 @@ const FeedResume = 1
 
 // FeedEventCallbackFunc performs mutation processing.  Return value indicates whether the mutation should trigger
 // checkpoint persistence (used to avoid recursive checkpoint document processing)
-type FeedEventCallbackFunc func(event FeedEvent) bool
+type FeedEventCallbackFunc func(ctx context.Context, event FeedEvent) bool
```

Comment:
Does the callback function need to be passed a context?  I would have expected that the callback itself would already have an established context, and doesn't need to have it passed back per event (e.g. there's already a context associated with the importListener, changeListener, etc).